docs/en/translation/index.rst

```diff
@@ -2,11 +2,14 @@
 Translation
 ***********
 
-The documentation is available in:
-
-- `English <https://tww.github.io/docs/en/>`_
-- `Deutsch <https://tww.github.io/docs/de/>`_
-- `Français <https://tww.github.io/docs/fr/>`_
-- `Italiano <https://tww.github.io/docs/it/>`_
-- `Română <https://tww.github.io/docs/ro/>`_
-- `Español <https://tww.github.io/docs/es/>`_
+The documentation is yet available in:
+
+- `English <https://teksi.github.io/wastewater/en/>`_
+- `Deutsch <https://teksi.github.io/wastewater/de/>`_
+- `Français <https://teksi.github.io/wastewater/fr/>`_
+
+It can be easily in other languages but please get in touch with us so you can contribute to the translation to other languages!
```

Comment:
missing translated
Suggested change: It can be easily translated in other languages, but please get first in touch with us before you contribute a translation in an other language!